acceptanceTests/tap_test.go

```diff
@@ -973,3 +973,197 @@ func TestTapDumpLogs(t *testing.T) {
 		return
 	}
 }
+
+func TestTapDaemon(t *testing.T) {
+	if testing.Short() {
+		t.Skip("ignored acceptance test")
+	}
+
+	tests := []int{50}
+
+	for _, entriesCount := range tests {
+		t.Run(fmt.Sprintf("%d", entriesCount), func(t *testing.T) {
+			cliPath, cliPathErr := getCliPath()
+			if cliPathErr != nil {
+				t.Errorf("failed to get cli path, err: %v", cliPathErr)
+				return
+			}
+
+			tapDaemonCmdArgs := getDefaultTapCommandArgsWithDaemonMode()
+
+			tapNamespace := getDefaultTapNamespace()
+			tapDaemonCmdArgs = append(tapDaemonCmdArgs, tapNamespace...)
+
+			tapCmd := exec.Command(cliPath, tapDaemonCmdArgs...)
+			t.Logf("running command: %v", tapCmd.String())
+
+			viewCmd := exec.Command(cliPath, getDefaultViewCommandArgs()...)
+
+			t.Cleanup(func() {
+				if err := runMizuClean(); err != nil {
+					t.Logf("error running mizu clean: %v", err)
+				}
+				if err := cleanupCommand(viewCmd); err != nil {
+					t.Logf("failed to cleanup view command, err: %v", err)
+				}
+			})
+
+			if err := tapCmd.Run(); err != nil {
+				t.Errorf("error occured while running the tap command, err: %v", err)
+				return
+			}
+
+			t.Logf("running command: %v", viewCmd.String())
+			if err := viewCmd.Start(); err != nil {
```

Comment:
do we wanna run the view command before the tappers are ready?

Reply:
we need view to be able to connect to the api server, you're right about the tappers tho, this might check entries too soon